lib/pure/os.nim

```diff
@@ -1688,13 +1688,28 @@ proc getFileInfo*(path: string, followSymlink = true): FileInfo =
     rawToFormalFileInfo(rawInfo, path, result)
 
 proc isHidden*(path: string): bool =
-  ## Determines whether a given path is hidden or not. Returns false if the
-  ## file doesn't exist. The given path must be accessible from the current
-  ## working directory of the program.
+  ## Determines whether the given ``path`` is hidden or not, using this reference https://en.wikipedia.org/wiki/Hidden_file_and_hidden_directory
```

Comment:
Keep to max line length of 80 characters please.

Reply:
doen